example_configs/maddy.md

```diff
@@ -0,0 +1,57 @@
+# Configuration for Maddy Mail Server
+
+Documentation for maddy LDAP can be found [here](https://maddy.email/reference/auth/ldap/).  
+Maddy will automatically create an imap-acct if a new user connects via LDAP.
+
+### URL
+Change to your LLDAP URL
+```
+urls ldap://lldap.example.com:3890
+```
+
+#### Docker
```

Comment:
No need for this section. People should know what the URL for their LLDAP server is, at least.